### PR TITLE
Update Russian month translations

### LINKED ISSUE #3035: Update values for Russian months
**Describe the bug**
Currently we are using the wrong set of months from the moment file, so Russian months look grammatically incorrect. We are using the standalone values while we want to use the format values instead.

`months : {
format:'января_февраля_марта_апреля_мая_июня_июля_августа_сентября_октября_ноября_декабря'.split('_'),
standalone:'январь_февраль_март_апрель_май_июнь_июль_август_сентябрь_октябрь_ноябрь_декабрь'.split('_')
}`

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://bbc.github.io/simorgh/?path=/story/moment-locales-editorial-view--russian-ru
2. See error

**Expected behaviour**
We need to switch to the other set of months in moment (`'января_февраля_марта_апреля_мая_июня_июля_августа_сентября_октября_ноября_декабря'.split('_')`) 
The months in https://bbc.github.io/simorgh/?path=/story/moment-locales-editorial-view--russian-ru  should match the correct values.

**Testing notes**
- [x] This bug fix is expected to need manual testing.

**Additional context**
Relevant JIRA ticket: WSFRONTPGS-238

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves #3035 

**Overall change:**
Override the values for `standalone` month translations to use `format` values instead.

**Code changes:**

- Add `months` object with `standalone` values to our updated Russian locale in `psammead-locales`
- Update tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 4.1.2 | [PR#???](https://github.com/bbc/psammead/pull/???) Update Russian month translations |
```
Comment (j-pendlebury) [inline packages/utilities/psammead-locales/CHANGELOG.md]:
```suggestion
| 4.1.2 | [PR#3064](https://github.com/bbc/psammead/pull/3064) Update Russian month translations |
```

**Review by EinsteinNjoroge — APPROVED** [review]
LGTM

**Review by DenisHdz — APPROVED** [review]
выглядит хорошо 👍

**Comment by PriyaKR** [comment]
Looks good http://localhost:8180/?path=/story/utilities-psammead-locales--moment-russian-ru seems to be using the format values.

**Pull request merged**